File: easy/problem136/solution.py
#!/usr/bin/python
# coding: utf-8 -*-
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ans = 0
        for x in nums:
            ans = ans^x
        return ans
nums = [1, 1, 2, 2, 3]
solution = Solution()
print(solution.singleNumber(nums))

File: easy/problem136/Solution.java
public class Solution {
    public int singleNumber(int[] nums){
        int result = 0;
        for (int i = 0; i < nums.length; i++){
            result ^= nums[i];
        }
        return result;
    }

    public static void main(String[] args){
        int[] input = {7, 3, 7, 9, 2, 9, 2};
        Solution solution = new Solution();
        System.out.println(solution.singleNumber(input));
    }
}